src/WebformCivicrmPostProcess.php

```diff
@@ -2085,7 +2085,9 @@ private function submitIPNPayment() {
       $paymentProcessor->doPayment($params);
     }
     catch (\Civi\Payment\Exception\PaymentProcessorException $e) {
-      drupal_set_message(ts('Payment approval failed with message: ') . $e->getMessage(),'error');
+        \Drupal::messenger()->addError(ts('Payment approval failed with message: %error ', [
+            '%error' =>  $e->getMessage(),
+        ]));
```

Comment:
@stesi561 Space fix required? should be 2 instead of 4 - https://www.drupal.org/docs/develop/standards/coding-standards#indenting

Reply:
Oops! Fixed.